src/context.cc

```diff
@@ -187,21 +187,21 @@ bool Context::has_local_variable(const std::string &name) const
  * @param what what is ignored
  * @param name name of the ignored object
  */
-static void print_ignore_warning(const char *what, const char *name, const Location &loc){
-	PRINTB("WARNING: Ignoring unknown %s '%s', %s.", what % name % loc.toString());
+static void print_ignore_warning(const char *what, const char *name, const Location &loc, const Context ctx){
```

Comment:
The Context should be a `&`, but I wonder if it's maybe cleaner to pass the documentPath as a `const string &` instead.